db/seeds/source_types.yml

```diff
@@ -4,6 +4,7 @@ amazon:
   :schema:
     :authentication:
     - :type: access_key_secret_key
+      :superkey_suported: true
```

Comment:
What about `:is_superkey`? Superkey_supported sounds more like an attribute of a source type.

Reply:
Sure, I'm ok with it since you'll be keying off of it more than me haha.